cfg_mgmt/bdba.py

```diff
@@ -0,0 +1,109 @@
+import datetime
+import logging
+
+import requests
+
+import cfg_mgmt
+import model.bdba
+
+logger = logging.getLogger(__name__)
+
+
+def get_api_key_expiry(
+    cfg_element: model.bdba.BDBAConfig
+) -> datetime.datetime:
+    credentials = cfg_element.credentials()
+    headers = {'Authorization': f'Bearer {credentials.token()}'}
+
+    response = requests.get(
```

Comment:
damn... I think it would be better to add this to delivery-service's `bdba.client`. However, this would introduce a cyclic dependency :-(

Could you:
- also add this into mentioned bdba-client
- add a reference to delivery-service-repository + a todo (to deduplicate)

OTOH, from python-package's pov, the dependency would not actually be cyclic.... sooo wdyt about declaring dependency against delivery-service's bdba-package?

/cc @zkdev | @8R0WNI3

Reply:
not too sure about introducing a dependency towards delivery-service's bdba-package ...
this would also contain recently introduced secret-mgmt.

in my opinion it is fine to keep this specific bdba implementation in parallel, as it is not to be expected that this will be expanded in future